README.md

```diff
@@ -2,20 +2,33 @@
 
 ## Introduction
 
-The `navigator.printing` Web API is a proposed standard for enabling printer-related functionality in web applications. Built on top of the Internet Printing Protocol (IPP), this API provides a set of JavaScript methods that allow developers to query local printers, submit print jobs, and manage print job options and status directly from web applications.
+The `navigator.printing` Web API is a proposed standard for enabling printer-related functionality in web applications. This API provides a set of JavaScript methods that allow developers to query local printers, submit print jobs, and manage print job options and status directly from web applications. To represent these concepts, it relies on the attribute names and semantics from the Internet Printing Protocol (IPP) specifications.
 
-Printing is a common and important task in many domains, including document management, label printing, receipt printing, and more. While web applications can generate printable content such as PDFs, images, and text documents, sending print jobs from web applications to local printers has traditionally required various workarounds such as relying on third-party plugins or polyfilling via `window.print()`; `navigator.printing` API aims to provide a standardized and seamless way for web applications to directly interact with printers, enabling developers to build printer-related features within their web applications.
+Printing is a common and important task in many domains, including document management, label printing, receipt printing, and more. While web applications can generate printable content such as PDFs, images, and text documents, sending print jobs from web applications to local printers has traditionally required various workarounds such as relying on third-party plugins or polyfilling via `window.print()`; the `navigator.printing` API aims to provide a standardized and seamless way for web applications to directly interact with printers, enabling developers to build printer-related features within their web applications.
```

Comment:
I would replace "sending print jobs" with "customizing a print job" because we need to acknowledge that web apps today can print documents just fine. The problem is that they can't tailor the document to the printer capabilities beyond what can be accomplished declaratively using CSS and they can't configure additional parameters such as copies and collation.